### DISCUSSION TITLE
Unable to Clear DAGs in Amazon Managed Workflows for Apache Airflow(MWAA)

### DISCUSSION THREAD

**Discussion by RiyazUlHaque** [apache/airflow#24589] [Q&A]
I have DAGs running into MWAA, but while I am trying to clear the DAGs it keeps on throwing the same error again and again.

**Error:**
oops! Something bad has happened. Please consider letting us know by creating a bug report using GitHub. 
Python version: 3.7.10 
Airflow version: 2.0.2 
Node: redact 

-------------------------------------------------------------------------------

 Error! Please contact server admin.
 
 Looking forward to a working solution to clear the DAGs.
 
 Thanks

**Comment by potiuk** [comment]
This can only be solved by MWAA support.
